files/templates/mbta/login/messages/messages_zh_CN.properties

```diff
@@ -42,14 +42,14 @@ invalidPasswordRegexPatternMessage=密码：无法匹配正则表达式模式。
 invalidPasswordHistoryMessage=密码：不得等于最后 {0} 个密码中的任何一个。
 invalidPasswordGenericMessage=密码：新密码与密码策略不匹配。
 
-SecondFactor.usernotFound=未找到要发送第二因子电子邮件的用户
-secondfactor.sendeMailerror=抱歉，发送第二因素电子邮件时出现问题
-secondfactor.validateCodeTitle=输入验证码
-secondfactor.code=输入电子邮件中的一次性验证码。
-secondfactor.bademailcode=一次性代码不正确
-secondfactor.usernoEmail=用户没有电子邮件
-secondfactor.dosubmit=登录
-secondfactor.codeExpired=代码已过期
+secondFactor.usernotFound=未找到要发送第二因子电子邮件的用户
+secondFactor.sendEmailerror=抱歉，发送第二因素电子邮件时出现问题
+secondFactor.validateCodeTitle=输入验证码
+secondFactor.code=输入电子邮件中的一次性验证码。
+secondFactor.bademailcode=一次性代码不正确
+secondFactor.usernoEmail=用户没有电子邮件
+secondFactor.dosubmit=登录
```

Comment:
question: a bunch of these look wrong (all lower-case). did we double check that they work?

Reply:
fixed in https://github.com/mbta/keycloak-deploy/commit/9f6315ec968af070bffadddb5650032f7af38f55, thanks